<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstantHead</class>
 <widget class="QWidget" name="ConstantHead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2,2,2">
   <item>
    <widget class="QLabel" name="lbl_name">
     <property name="text">
      <string>Constant Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="led_name"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_basicInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Basic Information</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
